test/integration/default/inspec/code_deploy_spec.rb

```diff
@@ -15,4 +15,22 @@
   describe file('/etc/cron.d/codedeploy-agent-update') do
     it { should_not exist }
   end
+
+  describe file('/etc/codedeploy-agent/conf/codedeployagent.yml') do
```

Comment:
Consider using the [`yaml`](https://www.inspec.io/docs/reference/resources/yaml/) resource.